app/views/users/show.html.erb

```diff
@@ -1,2 +1,47 @@
-<h1><%= @user.name %></h1>
-<p>Here will be the information of a specific user.</p>
+<div class="user-profile">
+
+  <!-- User information and profile picture -->
+  <div class="list">
+    <div class="user-image">
+      <img src="path_to_default_user_image.jpg" alt="User Image">
+    </div>
+    <div class="user-container">
+      <div class="user-name">
+        <%= @user.name %>
+        <p class="post-count">Number of posts: <%= @user.posts.count %></p>
+      </div>
+    </div>
+  </div>
+
+  <!-- User biography -->
+  <div class="user-bio">
+    <h1>Bio</h1>
+    <%= @user.bio %> <!-- Assuming 'bio' is the attribute for user's biography -->
+  </div>
+
+  <!-- Latest posts from the user -->
+  
+    <% @user.posts.limit(3).each do |post| %> 
```

Comment:
- [ ] Let's use the `recent_posts` model method in the user model to return the 3 most recent posts.

Reply:
Fix it: 
![image](https://github.com/lerfast/blog-app/assets/111823606/089d92dc-6918-4b68-b374-c2b9be203b21)